### PR TITLE
Update europe.yaml

### PR DESCRIPTION
**Pull request by ymollard**

### CODE REVIEW

File: data/europe.yaml [hunk 1]
```diff
@@ -549,3 +549,16 @@
...
+  lat: 44.806040
+  long: -0.605190
+  description: Engineering school in computer science, electronics, mechanics
+  link: http://robot-enseirb-matmeca.fr/
+- name: Inria
+  type: research institute
+  lat: 44.808479
+  long: -0.599851
+  description: Research institute for the digital sciences
+  link: https://www.inria.fr/en/centre/bordeaux
```
Comment (DLu) [inline data/europe.yaml]: Is there some way to distinguish this entry from the existing Inria one? 
```
- name: INRIA
  type: research institute
  lat: 45.217673
  long: 5.806614
```

Comment (ymollard) [reply data/europe.yaml]: Yes, this is done!

Comment (DLu) [reply data/europe.yaml]: Would it be safe to assume this one is Inria Bordeaux?

Comment (ymollard) [reply data/europe.yaml]: Definitely. Just wanted to go too fast :)

**Pull request merged**